model/flow/identity.go

```diff
@@ -183,8 +177,8 @@ type encodableIdentitySkeleton struct {
 }
 
 type encodableIdentity struct {
-	encodableIdentitySkeleton
-	ParticipationStatus string
+	EncodableIdentitySkeleton encodableIdentitySkeleton
```

Comment:
Good catch, this is needed to implement proper RLP encoding. Previously the identity skeleton was not part of the RLP serialization.